pages/forms/styles.css

```diff
@@ -0,0 +1,157 @@
+
+footer a {
+  color: var(--primary-color);
+}
+
+a> :last-child {
+  border-bottom: 0.2px var(--text-color) solid;
+}
+
+header,
+.header {
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.header a{
+  color: var(--primary-color);
+}
+
+/* 
+a > div > h5 {
+  float: right;
+} */
+
+/* a > a:hover {
+  border-top: 1px var(--text-color) solid;
+} */
+
+* {
+  color: var(--text-color);
+  text-decoration: none;
+}
+
+body {
+  background-image: url("background.png");
+}
+
+.header {
+  float: right;
+  text-align: right;
+}
+
+h1 {
+  /* display: block; */
+  float: left;
+  margin-top: 0;
+}
+
+/* * body {
+  align-items: center;
+  color: green;
+  font-size: 16px;
+  height: 100vh;
+  margin: 0;
+  width: 100%;
+}
+
+header,
+.home-page,
+footer {
+  background-color: white;
+  margin: 0 auto;
+  max-width: 630px;
+  min-width: 768px;
+}
+
+h1  {
+  text-align: center;
+  margin-top: 2.5rem;
+  color: aqua;
+
+}
+
+.header * {
+  display: grid;
+  margin-bottom: 0;
+  padding:25px;
+} */
+
+body > 
+header, 
+.header, 
+.home-page {
+  background-color: white;
+  color: var(--text-color);
+  font-family: var(--dont-family);
+  margin: 0 auto;
+  margin: auto;
+  max-width: 1200px;
+}
+
+body>
+header, 
+.header {
+  padding: 0 25px;
+}
+
+.home-page {
+  padding: 25px;
+}
+
+/* * {
+  transition: all 0.3s;
+} */
+
+:root {
```

Comment:
I would move this to the top of the file and follow this order:
- variables
- general/global elements (*, HTML, body, h1, h2, span, etc)
- general classes or IDs
- page specific styles